.circleci/config.yml

```
@@ -342,9 +345,122 @@ jobs:
                 include_job_number_field: false
                 include_project_field: false
                 failure_message: ':red_circle: WordPress Android connected tests failed on \`${CIRCLE_BRANCH}\` branch after merge by ${CIRCLE_USERNAME}. See <https://console.firebase.google.com/u/0/project/api-project-108380595987/testlab/histories/bh.e0c3a59bd9ed670|Firebase console test results> for details.\n\nPlease reach out in #platform9 if you think this failure is not caused by your changes, so we can investigate.'
+  raw-screenshots:
+    executor: android-jdk8
+    working_directory: /tmp/workspace
+    steps:
+      - git/shallow-checkout:
+          init-submodules: true
+      - checkout-submodules
+      - bundle-install/bundle-install:
+          cache_key_prefix: v1-raw-screenshots
+      - android/restore-gradle-cache
+      - copy-gradle-properties
+      - update-gradle-memory
+      - restore-gutenberg-bundle-cache
+      - run:
+          name: Ensure assets folder exists
+          command: mkdir -p libs/gutenberg-mobile/gutenberg/packages/react-native-bridge/android/src/main/assets
+      - attach_workspace:
+          at: libs/gutenberg-mobile/gutenberg/packages/react-native-bridge/android/src/main/assets
+      - run: 
+          name: Setup assets
+          command: |
+            INDEX_BUNDLE_PATH="libs/gutenberg-mobile/gutenberg/packages/react-native-bridge/android/build/assets/index.android.bundle"
+            mkdir -p "$(dirname "${CIRCLE_WORKING_DIRECTORY}/${INDEX_BUNDLE_PATH}")"
+            cp ~/project/"${INDEX_BUNDLE_PATH}" "${CIRCLE_WORKING_DIRECTORY}/${INDEX_BUNDLE_PATH}"
+      - run:
+          name: Build
+          environment:
+            SUPPRESS_GUTENBERG_MOBILE_JS_BUNDLE_BUILD: 1
+          command: ./gradlew WordPress:assembleVanillaDebug WordPress:assembleVanillaDebugAndroidTest --stacktrace
+      - run:
+          name: Decrypt credentials
+          command: openssl aes-256-cbc -md sha256 -d -in .circleci/.firebase.secrets.json.enc -out .circleci/.firebase.secrets.json -k "${FIREBASE_SECRETS_ENCRYPTION_KEY}"
+      - android/firebase-test:
+          key-file: .circleci/.firebase.secrets.json
+          type: instrumentation
+          apk-path: WordPress/build/outputs/apk/vanilla/debug/org.wordpress.android-vanilla-debug.apk
+          test-apk-path: WordPress/build/outputs/apk/androidTest/vanilla/debug/org.wordpress.android-vanilla-debug-androidTest.apk
+          test-targets: package org.wordpress.android.ui.screenshots
+          # NOTE: When updating this list of devices, ensure to also update the `phone:` and `tenInch:` parameters in "Download Raw Screenshots from Google Storage"
+          devices: |
+            model=blueline,version=28,orientation=portrait
+            model=gts3lltevzw,version=28,orientation=default
```

After seeing @jkmassel's comment on screenshots being stretched when rendered in promo material, we might want to enforce the right orientation here

```suggestion
            model=gts3lltevzw,version=28,orientation=landscape
```